go/dhcp/main.go

```diff
@@ -608,3 +673,11 @@ func (h *Interface) ServeDHCP(ctx context.Context, p dhcp.Packet, msgType dhcp.M
 	return answer
 
 }
+
+func recoverName(options dhcp.Options) {
+	if r := recover(); r != nil {
+		fmt.Println("recovered from ", r)
+		fmt.Println(errors.Wrap(r, 2).ErrorStack())
+		spew.Dump(options)
```

Comment:
Is debugging needed here?

Reply:
It's a way to see which dhcp packet cause the crash.